### PR TITLE
actionified, formatted and doxy'd

### PR DESCRIPTION
**Pull request by siddacious**

### CODE REVIEW

**Comment by siddacious** [comment]
@ladyada I believe this is failing be cause `build_platform.py` doesn't check for `.test.skip` here
https://github.com/adafruit/ci-arduino/blob/master/build_platform.py#L181

should I put in a PR to add that?

**Comment by ladyada** [comment]
i dont think thats correct? something isnt compiling in general. take a look later

**Comment by siddacious** [comment]
@ladyada I pushed a change that allows the failing test to compile, though I'd like your input on the 'solution' of putting the `DVOLUME` check in `create`

**Comment by siddacious** [comment]
This seems to be stuck. I'll try giving it a bump

**Comment by siddacious** [comment]
@ladyada another gtg

File: .github/workflows/githubci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - name: test platforms
+      run: python3 ci/build_platform.py main_platforms
+
+    - name: clang
+      run: python3 ci/run-clang-format.py -e "ci/*" -e "bin/*" -r . 
+
+    - name: doxygen
+      env:
+        GH_REPO_TOKEN: ${{ secrets.GH_REPO_TOKEN }}
+        PRETTYNAME : "Adafruit FT6206 Arduino Library"
```
Comment (ladyada) [inline .github/workflows/githubci.yml]: fix

**Review by ladyada — APPROVED** [review]

**Pull request merged**